public/static/docs/command-reference/import.md

```diff
@@ -201,7 +202,7 @@ $ dvc import git@github.com:iterative/dataset-registry.git \
 ```
 
 `dvc import` provides a better way to incorporate data files tracked in external
-<abbr>DVC repositories</abbr> because it saves the connection between the
+<abbr>Git repositories</abbr> because it saves the connection between the
```

Comment:
we should probably still mention DVC ... Git/DVC repositories? let's come with some simple way to describe this cc @jorgeorpinel

Reply:
In the context of this example this paragraph was correct. No need for this change.

Reply:
Still pending but will resolve this and reopen on the current version for readability.